types/dwt/Addon.OCR.d.ts

```diff
@@ -6,8 +6,8 @@ export interface OCR {
      * @param path The URL to download the add-on (typically a ZIP file).
      * @param successCallback A callback function that is executed if the request succeeds.
      * @param failureCallback A callback function that is executed if the request fails.
-     * @argument errorCode The error code.
-     * @argument errorString The error string.
+     * @param errorCode The error code.
+     * @param errorString The error string.
```

Comment:
Hmm, also unintentional, should be reverted.

```suggestion
     * @argument errorCode The error code.
     * @argument errorString The error string.
```